src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,95 @@
-import { useState } from 'react';
+import { useState, FormEvent } from 'react';
+
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
+
+type AddNewMovieProp = {
+  onAdd: (param: Movie) => void;
+};
 
-export const NewMovie = () => {
+export const NewMovie: React.FC<AddNewMovieProp> = ({ onAdd }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, SetCount] = useState(0);
```

Comment:
it should start with lowercase for all of them
```suggestion
  const [count, setCount] = useState(0);
```